src/main/java/net/sf/mzmine/main/MZmineCore.java

```diff
@@ -59,6 +60,13 @@
  */
 public final class MZmineCore {
 
+	public static final String MZmineName      = "MZmine PeakInvestigator™ Edition";
```

Comment:
The ```MZmineName``` and related variables should probably not make it into the push to the upstream project. Can you look into whether this info can be exposed via ```pom.xml``` and the Maven build system? I believe you can have a properties file in ```src/main/resources``` and the following in the POM:

```
        <resources>
                <resource>
                        <directory>src/main/resources</directory>
                        <filtering>true</filtering>
                </resource>
        </resources>
```